packages/transformers/clear/src/ClearTransformer.js

```diff
@@ -0,0 +1,10 @@
+// @flow strict-local
+
+import {Transformer} from '@parcel/plugin';
+
+export default (new Transformer({
+  transform({asset}) {
+    asset.symbols.clear();
```

Comment:
Can you give us some context about why this plugin is necessary?

Reply:
@mischnic perhaps you could speak to this?

Reply:
I'm not really sure how `this.#value.symbols` is used, the worklet or worker will likely not be able to share dependencies with the main javascript thread (for now) so I'm curious if this is an attempt to prevent the asset from having additional globals or other dependencies incorrectly bundled into the asset during packaging?

Reply:
This is purely to workaround the `foo.js does not export 'default'` error when doing `import x from "worker:./foo.js"`, has nothing to do with shared bundles.

An alternative is special-casing `asset.isIsolated` in the symbol propagation logic itself.